lib/common/schemaUtilsCommon.js

```diff
@@ -2,7 +2,207 @@
  * This file contains util functions that are common between versions
  */
 
-const parse = require('../parse.js');
+const parse = require('../parse.js'),
+  _ = require('lodash'),
+  typesMap = {
+    integer: {
+      int32: '<integer>',
+      int64: '<long>'
+    },
+    number: {
+      float: '<float>',
+      double: '<double>'
+    },
+    string: {
+      byte: '<byte>',
+      binary: '<binary>',
+      date: '<date>',
+      'date-time': '<dateTime>',
+      password: '<password>'
+    },
+    boolean: '<boolean>',
+    array: '<array>',
+    object: '<object>'
+  },
+
+  /* eslint-disable arrow-body-style */
+  schemaTypeToJsValidator = {
+    'string': (d) => typeof d === 'string',
+    'number': (d) => !isNaN(d),
+    'integer': (d) => !isNaN(d) && Number.isInteger(Number(d)),
+    'boolean': (d) => _.isBoolean(d) || d === 'true' || d === 'false',
```

Comment:
@harshit-budhraja This is existing logic and was present before as well under `lib/schemaUtils.js`.

Reply:
@VShingala But logically, I still see that `|| d === 'true' || d === 'false'` is something that is not required. It would be good to remove it to improve code quality. Also, we already have tests in place to check for regression breakages (if any)
Thoughts?

Reply:
@VShingala Oh my bad! I wasn't aware we expect string values for this. Looking at the property name `boolean`, it suggested that we're only expected to handle for boolean.

Anyway, this works then 👍🏻